sonic_data_client/db_client.go

```diff
@@ -91,18 +91,19 @@ func (val Value) Compare(other queue.Item) int {
 }
 
 type DbClient struct {
-	prefix  *gnmipb.Path
+	prefix	*gnmipb.Path
 	pathG2S map[*gnmipb.Path][]tablePath
-	q       *queue.PriorityQueue
+
+	q		*queue.PriorityQueue
 	channel chan struct{}
 
 	synced sync.WaitGroup  // Control when to send gNMI sync_response
-	w      *sync.WaitGroup // wait for all sub go routines to finish
-	mu     sync.RWMutex    // Mutex for data protection among routines for DbClient
+	w	   *sync.WaitGroup // wait for all sub go routines to finish
```

Comment:
Why change this indenting? Tab seems to be too far.

Reply:
I'm not really sure what changed, but the file was run through gofmt, so this should be the official formatting preference for golang in the latest commit pushed to this PR.